xml/System.Security.Cryptography/AesManaged.xml

```diff
@@ -490,7 +494,11 @@
       <Docs>
         <summary>Gets or sets the initialization vector (IV) to use for the symmetric algorithm.</summary>
         <value>The initialization vector to use for the symmetric algorithm</value>
-        <remarks>To be added.</remarks>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The symmetric encryption algorithms operating in Cipher Block Chaining (CBC) mode combine the block with the portion of data from the previous block in the process of encryption. When encrypting the first block, initialization vector is used instead of the previous block value. The initialization vector size must be the same as the block size.  
```

Comment:
Seems like this should copy, or point to, SymmetricAlgorithm's definition of this.